src/utils/data-mapper.ts

```diff
@@ -209,7 +141,40 @@ const tmdbCollectionMap = {
 const tmdbMovieMap = {
   'belongs_to_collection.id': 'collectionTmdbID',
   'budget': 'budget',
-  'credits': 'credits',
+  'credits': [
+    { key: 'credits' },
+    {
+      key: 'actors',
+      transform: (credits: CreditsResponse | EpisodeCreditsResponse): string[] => {
+        // populate the old "actors" array which came from OpenSubtitles
+        if (!credits?.cast) {
+          return [];
+        }
+        const actors = [];
+        credits.cast.forEach((castEntry: Cast) => {
+          actors.push(castEntry.name);
+        });
+        return actors;
+      },
+    },
+    {
+      key: 'directors',
+      transform: (credits: CreditsResponse | EpisodeCreditsResponse): string[] => {
+        // populate the old "directors" array which came from OpenSubtitles
+        if (!credits?.crew) {
+          return [];
+        }
+
+        const directors = [];
+        credits.crew.forEach((crewEntry: Crew) => {
+          if (crewEntry.job === 'Director') {
```

Comment:
here you may have a lot of secondary directors.
you may filter a bit more like in java :

`if ("Directing".equals(person.getDepartment()) && "Director".equals(person.getJob())) {`